env0/resource_environment_test.go

```diff
@@ -2437,6 +2438,7 @@ func TestUnitEnvironmentWithSubEnvironment(t *testing.T) {
 				Revision:             subEnvironment.Revision,
 				Workspace:            updatedSubEnvironment.Workspace,
 				ConfigurationChanges: updatedSubEnvironment.Configuration,
+				UserRequiresApproval: true,
```

Comment:
`false` for 'approve_plan_automatically' will set `UserRequiresApproval` to `true`